src/components/SearchBox.vue

```diff
@@ -56,6 +56,10 @@ export default {
     ...mapActions('search', ['doSearch']),
     submitSearch: function () {
       this.doSearch(this.searchQuery)
+    },
+    submitMobileSearch: function () {
```

Comment:
I agree that this is not the most elegant way to do this... I do not have a better idea right now.